### ISSUE TITLE
请问baseline 有没有用到 video_duration

### ISSUE THREAD

**Issue by wzhang1** [shenweichen/Bytedance_ICME2019_challenge_baseline#4]
在文档里提到没有， train.py 里面好像有

**Comment by shenweichen** [comment]
train.py里有，但是我注释掉了，所以实际运行的时候没有用video

**Comment by wzhang1** [comment]
> train.py里有，但是我注释掉了，所以实际运行的时候没有用video

谢谢

**Comment by wzhang1** [comment]
> train.py里有，但是我注释掉了，所以实际运行的时候没有用video

请问完全不改代码，提交结果大约0.75， 是因为video_duration overfit吗

**Comment by shenweichen** [comment]
没明白您的意思,baseline的代码直接提交大概是0.71，怎么会0.75呢
另外，baseline中用到了video_duration

**Comment by wzhang1** [comment]
> 没明白您的意思,baseline的代码直接提交大概是0.71，怎么会0.75呢
> 另外，baseline中用到了video_duration


0.75279626640149 | track2: f, l = 0.683057590028, 0.915519844606

**Comment by wzhang1** [comment]
> 没明白您的意思,baseline的代码直接提交大概是0.71，怎么会0.75呢
> 另外，baseline中用到了video_duration

VALIDATION_FRAC = 0.2
local validation:
15697872/15697872 [==============================] - 434s 28us/step - loss: 0.6232 - finish_loss: 0.5737 - like_loss: 0.0352 - val_loss: 0.6381 - val_finish_loss: 0.5846 - val_like_loss: 0.0364
finish auc
0.727005452958114
like auc
0.9349380502656399

**Comment by shenweichen** [comment]
直接看提交的结果把，线下我也忘记多少分了，提交的结果是包含video_duration这个字段的
